javaScript/speaker-data.js

```diff
@@ -26,15 +26,15 @@ const speakerFour = {
 const speakerDataArr = [speakerOne, speakerTwo, speakerThree, speakerFour];
```

Comment:
- [ ] [REQUIRED] Good job so far! Please keep original layout desigin [provided by Microverse](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) so add `2` speaker to your `Featured Speakers` part because all speakers should equal to `6`. Also make sure their `distance`, they seem stuck together 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/217559157-441860c6-89be-4636-8944-706f026ebe68.png)